mahta/CSS/CSS basics/index2.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
As I mentioned earlier, based on the issue, you need to add class to this tag, not the body tag.